src/bar.js

```diff
@@ -186,15 +190,64 @@ export default class Bar {
             }
 
             this.gantt.unselect_all();
-            this.group.classList.toggle('active');
+            
+            //  add dependency by popup button
+            if(this.gantt.dependencyBar != null){
```

Comment:
Please use snake case: `dependency_bar`. Please do it everywhere you have done your changes.